internal/common/runtime.go

```diff
@@ -0,0 +1,36 @@
+package common
+
+import "runtime/debug"
+
+var (
+	// Git SHA commit (first 4 characters)
+	BuildCommit string
+
+	// Build date and time
+	BuildTime string
+
+	// BuildGoVersion carries Go version the binary was built with
+	BuildGoVersion string
+)
+
+func init() {
+	bi, ok := debug.ReadBuildInfo()
+
+	if !ok {
+		BuildTime = "N/A"
+		BuildCommit = "HEAD"
+	}
+
+	BuildGoVersion = bi.GoVersion
```

Comment:
Is `bi` actually defined (i.e. non-nil) if we get a `!ok` before (if not we would crash here)? I wonder if it wouldn't be cleaner to define the defaults directly in `var BuildCommit = "N/A"` and just return on `if !ok { return }`(?)